device/bgp/bgpsrtebindingsubtlv.yaml

```diff
@@ -2,12 +2,12 @@ components:
   schemas:
     BgpSrte.BindingSubTlv:
       description: >-
-        Configuration for BGP binding sub TLV settings.
-      type: object
+        Configuration for the binding SID sub-TLV is used to signal 
+        the binding SID related information of the SR Policy candidate path.      type: object
       properties:
         binding_sid_type:
           description: >-
-            TBD
+            A user to specify one of the No Binding SID, Four Octets Sid or IPv6 SID.
```

Comment:
Type of the binding SID. Supported types are "No Binding SID" or "Four Octets Sid" or "IPv6 SID".